components/Footer.tsx

```diff
@@ -1,6 +1,8 @@
 import styles from './Footer.module.scss';
 import { FunctionComponent } from 'react';
 import Link from '../components/Link';
+import Image from 'next/image';
+import FaqSvg from '../public/images/svg/faq.svg';
```

Comment:
How comes you're importing the file like this instead of just adding the reference to the public URL to src attribute?

Reply:
Ach, it doesn’t really matter I guess. I was just testing out if this had something to do with the hydration error, but turns out it was ok.